src/Components/About/About.js

```diff
@@ -4,10 +4,25 @@ import './About.css'
 
 const About = () => {
   return (
-    <div>
+    <section className='main-section'>
       <Header />
-      <h1>about me</h1>
-    </div>
+      <header className='about-me-header'>
+        <h1 className='about-header'>About Me</h1>
+      </header>
+      <article className='my-player-card'>
+        <div className='my-image'>
+          <img src="" alt="Image of app developer"></img>
```

Comment:
Wasn't sure if this was a placeholder for now but the image didn't appear for me on my end